stable_plugins/workflow/workflows/templates/workflow_human_task.html

```diff
@@ -37,22 +51,158 @@
+                console.log(camForm)
+            }
+        });
+
+        document.querySelector("button.qhana-form-submit").addEventListener("click", (event) => {
+            event.preventDefault();
+            afterFormSubmit();
+        });
+
+        function getFormVariables() {
+            const formData = new FormData();
+            if (!camundaFormInstance) {
+                return formData;
+            }
+
+            camundaFormInstance.retrieveVariables();
+
+            // content from https://github.com/camunda/camunda-bpm-platform/blob/cb15a424bcceb920e1ea0b199c6759bc886cbbc4/webapps/frontend/camunda-bpm-sdk-js/lib/forms/camunda-form.js#L699
+
+            var varManager = camundaFormInstance.variableManager;
+            var vars = varManager.variables;
+
+            // The default display value is different from the original value in varManager
+            camundaFormInstance.fields.forEach(function(field) {
+                if (vars[field.variableName]) {
+                vars[field.variableName].defaultValue = field.originalValue;
+
+                if (
+                    field.originalValue === '' ||
+                    typeof field.originalValue === 'undefined'
+                ) {
+                    vars[field.variableName].defaultValue = vars[field.variableName].value;
+                }
+                }
+            });
+
+            for (var v in vars) {
+                // only submit dirty variables
+                // LIMITATION: dirty checking is not performed for complex object variables
+
+                var val = vars[v].value;
+
+                // We want implicit type conversion in this case, the defaultValue is always a string
+                if (varManager.isDirty(v) || vars[v].defaultValue != val) {
+                    // if variable is JSON, serialize
+                    if (varManager.isJsonVariable(v)) {
+                        val = JSON.stringify(val);
+                    }
+
+                    // if variable is Date, add timezone info
+                    if (val && varManager.isDateVariable(v)) {
+                        val = moment(val, moment.ISO_8601).format('YYYY-MM-DDTHH:mm:ss.SSSZZ');
+                    }
+
+                    formData.set(v, val);
+                }
+            }
+            return formData;
+        }
+
+        function afterFormSubmit() {
+            const formData = getFormVariables();
+            sendMessage("ui-loading");
+
+            submitFormData(formData, processUrl, "post")
+                .then(
+                    (response) => {
+                        if (response.status === 200) {
+                            sendMessage({
+                                type: "form-submit",
+                                formData: (new URLSearchParams(formData)).toString(),
+                                formDataType: "application/x-www-form-urlencoded",
+                                dataInputs: [],  // TODO
```

Comment:
todo comment

Reply:
This is a hard problem, and it is unlikely that an external form even takes QHAna data inputs. So I would just leave this as a TODO.